pkgs/development/libraries/google-cloud-cpp/default.nix

```diff
@@ -45,6 +47,9 @@ stdenv.mkDerivation rec {
     cmake
     ninja
     pkg-config
+  ] ++ lib.optionals (!doInstallCheck) [
```

Comment:
Shouldn't this be `(doInstallCheck)`?

Reply:
Added a comment about this. Does it answer your question?

Reply:
yes